content/docs/concepts/traffic-management/index.md

```diff
@@ -1053,43 +1000,16 @@ rules to:
   [multicluster Istio mesh](/docs/tasks/multicluster/gateways/#configure-the-example-services)
   on Kubernetes.
 
-You don’t need to add a service entry for every mesh-external service that you
+You don’t need to add a service entry for every external service that you
 want your mesh services to use. By default, Istio configures the Envoy proxies
-to passthrough requests from unknown services. You can also use service entries
-to configure internal infrastructure:
+to passthrough requests to unknown services, although you can't use Istio features
+to control the traffic to destinations that are not registered in the mesh.
 
-- A **mesh-internal** service entry adds a service running in the mesh, which
-   doesn't have a service discovery adapter that would add it to the abstract
-   model automatically.
-
-    For example, you create mesh-internal service entries for the services
-    running on VMs outside the Kubernetes cluster but within the network.
-    Mutual TLS authentication (mutual TLS) is enabled by default for mesh-internal
-    service entries, but to change the authentication method, you can configure
-    a destination rule for the service entry.
-
-- A **mesh-external** service entry adds a service without and Envoy proxy to
-   the mesh. You configure a mesh-external service entry so that a service
-   inside the mesh can make API calls to an external server. You can use
-   service entries with an egress gateway to ensure all external services are
-   accessed through a single exit point.
-
-    mutual TLS is disabled by default for mesh-external service entries, but you can
-    change the authentication method by configuring a destination rule for the
-    service entry. Because the destination is external to the mesh, the Envoy
-    proxies of the services inside the mesh enforce the configured policies for
-    services added through mesh-external service entries.
-
-You can use mesh-external service entries to perform the following
```

Comment:
This list aims to answer the following user questions: What do I use service entries for? Why should I use service entries? Please add TLS origination and single exit point to the list but don't get rid of the others. AFAIK, there are users that employ service entries to perform the configurations currently listed and we keep them in the document.

Reply:
The first two items are totally arbitrary, and the second one is an overly technical detail that is meaningless without further elaboration, which woul not be appropriate here.